### ISSUE TITLE
splitpane_divider color

### ISSUE THREAD

**Issue by joggienl** [henrikdahl/hyperterm-chesterish#8]
Hello,

I would like to suggest a different color for the split panes. The borders are now almost invisible. I've chosen for the foregroundColor als color, but it can definitely be another one.

My suggestion would be to add the following to the config:

```js
// Config
exports.decorateConfig = config => {
    return Object.assign({}, config, {
        ...,
        css: `
            ${config.css || ''}
            ::selection {
                background: #9198A2 !important;
            }
            .splitpane_divider {
                background-color: ${foregroundColor};
            }
        `
    });
};
```

**Comment by henrikruscon** [comment]
Thanks for your suggestion.

I feel like this is more of a border issue considering it'll look odd if the border and divider don't match. So I just pushed a version with a lighter border.

Hope that works for you ✌️

**Comment by joggienl** [comment]
Do you mean this commit https://github.com/henrikdahl/hyper-chesterish/commit/0c3188ec45117edc0e7eda9837859365ad0e30cd ?

That actually made the border even darker not lighter.

**Comment by henrikruscon** [comment]
@joggienl 

Commit message is wrong, the color is indeed lighter, you can compare them [here](http://www.colorcombos.com/combotester.html?color0=2d3847&color1=323e4d&show_text=)

Left is old color, right is new one ✌️

**Comment by joggienl** [comment]
I see indeed, I did not compare the actual colors itself ;-)

For me the border and thus divider color still doesn't stand out enough. I do notice it little better on my retina screen, but on my non-retina screen it still is almost invincible.

The border color goes for windows/tabs and the split divider (as far as I know). For the window and tab this goes fine, it's a subtle difference which makes it look pretty. The border in this instance is not something you need to interact with (no dragging or so with the border).

The divider on the other hand _is_ actually actionable. Thats why I made it way more white in my PR proposal: you need to be able to interact with it. Which is hard if you can't see it.

So, in short, do you see possibilities to make it even more lighter?

Aside from that it might also be nice to extend the example images with an image with some split panes in :-)

🤞

**Comment by henrikruscon** [comment]
I love playing with a narrow contrast color palette with everything I design. I'm also doing it on a Retina display with 500 nits of brightness.

I'm sure it might look very different on older displays.

I think you make a solid argument and I'll look into options for improving the split divider today ✌️

**Comment by joggienl** [comment]
Well, to be honest, this "narrow contrast color palette" is what makes me want to use this scheme so much :-)

Thanks in advance, curious to see the results!

**Comment by joggienl** [comment]
Hey hello!

Curious, have you had any time for this yet?
I was also thinking what might be good addition for this: make the active pane "jump" out (or give the inactive ones a different color...

Not sure if that will fit in the theme, but it could be a nice addition..
